geometry_msgs/Pose position
uint8 sem_status
float64 time_to_change
